helix-core/src/main/java/org/apache/helix/manager/zk/ParticipantManager.java

```diff
@@ -77,14 +76,17 @@
   final LiveInstanceInfoProvider _liveInstanceInfoProvider;
   final List<PreConnectCallback> _preConnectCallbacks;
 
+  private final String _sessionId;
```

Comment:
This is probably OK, but one minor concern is none backward compatible that you change this field to private.